content/ko/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -0,0 +1,761 @@
+
+```shell
+kubectl describe pod frontend | grep -A 3 Events
+```
+```
+Events:
+  FirstSeen LastSeen   Count  From          Subobject   PathReason      Message
+  36s   5s     6      {scheduler }              FailedScheduling  Failed for reason PodExceedsFreeCPU and possibly others
+```
+
+위의 예에서, 노드의 CPU 리소스가 충분하지 않아 이름이
+"frontend"인 파드를 스케줄하지 못했다. 메모리 부족(PodExceedsFreeMemory)으로
+인해 유사한 오류 메시지가 장애를 알릴 수도 있다. 일반적으로, 파드가
+이 타입의 메시지로 보류 중인 경우, 몇 가지 시도해 볼 것들이 있다.
+
+- 클러스터에 더 많은 노드를 추가한다.
+- 불필요한 파드를 종료하여 보류 중인 파드를 위한 공간을 확보한다.
+- 파드가 모든 노드보다 크지 않은지 확인한다. 예를 들어, 모든
+  노드의 용량이 `cpu: 1` 인 경우, `cpu: 1.1` 요청이 있는 파드는
+  절대 스케줄되지 않는다.
+
+`kubectl describe nodes` 명령으로 노드 용량과 할당된 양을
+확인할 수 있다. 예를 들면, 다음과 같다.
+
+```shell
+kubectl describe nodes e2e-test-node-pool-4lw4
+```
+```
+Name:            e2e-test-node-pool-4lw4
+[ ... 명확하게 하기 위해 라인들을 제거함 ...]
+Capacity:
+ cpu:                               2
+ memory:                            7679792Ki
+ pods:                              110
+Allocatable:
+ cpu:                               1800m
+ memory:                            7474992Ki
+ pods:                              110
+[ ... 명확하게 하기 위해 라인들을 제거함 ...]
+Non-terminated Pods:        (5 in total)
+  Namespace    Name                                  CPU Requests  CPU Limits  Memory Requests  Memory Limits
+  ---------    ----                                  ------------  ----------  ---------------  -------------
+  kube-system  fluentd-gcp-v1.38-28bv1               100m (5%)     0 (0%)      200Mi (2%)       200Mi (2%)
+  kube-system  kube-dns-3297075139-61lj3             260m (13%)    0 (0%)      100Mi (1%)       170Mi (2%)
+  kube-system  kube-proxy-e2e-test-...               100m (5%)     0 (0%)      0 (0%)           0 (0%)
+  kube-system  monitoring-influxdb-grafana-v4-z1m12  200m (10%)    200m (10%)  600Mi (8%)       600Mi (8%)
+  kube-system  node-problem-detector-v0.1-fj7m3      20m (1%)      200m (10%)  20Mi (0%)        100Mi (1%)
+Allocated resources:
+  (Total limits may be over 100 percent, i.e., overcommitted.)
+  CPU Requests    CPU Limits    Memory Requests    Memory Limits
+  ------------    ----------    ---------------    -------------
+  680m (34%)      400m (20%)    920Mi (12%)        1070Mi (14%)
+```
+
+위의 출력에서, ​파드가 1120m 이상의 CPU 또는 6.23Gi의 메모리를
+요청하면, 노드에 맞지 않음을 알 수 있다.
+
+`Pods` 섹션을 살펴보면, 파드가 노드에서 공간을 차지하는 것을
+볼 수 있다.
+
+시스템 데몬이 사용 가능한 리소스의 일부를 사용하기 때문에, 파드에
+사용 가능한 리소스의 양이 노드 용량보다 적다. `allocatable` 필드
+[NodeStatus](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodestatus-v1-core)는
+파드가 사용할 수 있는 리소스의 양을 제공한다. 자세한 정보는
+[노드 할당 가능 리소스](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md)를 참조한다.
+
+[리소스 쿼터](/ko/docs/concepts/policy/resource-quotas/) 기능은
+소비될 수 있는 리소스의 총량을 제한하도록 구성할 수 있다. 네임스페이스와
+함께 사용하면, 한 팀이 모든 리소스를 사용하지 못하게 할 수 있다.
+
+### 내 컨테이너가 종료되었다
+
+리소스가 부족하여 컨테이너가 종료될 수 있다. 리소스
+제한에 도달하여 컨테이너가 종료되고 있는지 확인하려면,
+관심있는 파드에서 `kubectl describe pod` 를 호출한다.
```

Comment:
```suggestion
관심있는 파드로 `kubectl describe pod` 를 호출한다.
```

Reply:
> Your Container might get terminated because it is resource-starved. To check
> whether a Container is being killed because it is hitting a resource limit, call
> `kubectl describe pod` on the Pod of interest:

원문에 맞게 `관심있는 파드에서` 가 더 적당하지 않을까요?

Reply:
`kubectl describe pod`의 대상으로 쓰인 것이라, `파드에 대해` 가 좀 더 적당해 보입니다.

Reply:
그렇네요. 반영하겠습니다.